.idea/modules/common/DesignCompose.common.main.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
copyright here?

Reply:
As far as I can tell, copyright headers are only required for source, documentation and build-related files. (go/copyright) I can't find anything about code editor configuration files, and it says that generated files don't need it. I'd say this isn't source, documentation or build-related, since this file won't change how the code is built, and it was generated by Android Studio.

Also, the `addlicense` tool, which was written by a Googler and is recommended by [go/opensource](go/releasing/preparing#license-headers) isn't trying to add it. This is probably because the file's extension is `.iml`. `addlicense` seems to work off of file extensions, which would be why it's adding the license to all of the `.xml` files, but I don't think that it strictly needs to.

I could either try to get `addlicense` to start including `.iml` files or just leave them alone, and I'm inclined to just leave them alone.